src/adminPartMain/kotlin/AgentState.kt

```diff
@@ -36,6 +36,8 @@ class AgentState(
             _lastBuildCoverage.value = value
         }
 
+    fun prevBuildVersion() = prevState?.agentInfo?.buildVersion ?: ""
```

Comment:
the correct previous build version is already stored as ClassesData field. you're going to get the **previous state** build version this way. is that what you intended to do?